pkg/apiserver/etcd_config.go

```diff
@@ -88,6 +88,8 @@ func (c completedEtcdConfig) NewServer() (*ServiceCatalogAPIServer, error) {
 
 	roFactory := etcdRESTOptionsFactory{
```

Comment:
can use `apiserver/pkg/server/options#SimpleRestOptionsFactory` instead. can be a follow-up, though it would be good to unify prior to having requirements for data migration